lib/index.js

```diff
@@ -39,5 +39,36 @@ function purify(htmlInput, options, cb) {
   });
 }
 
+/**
+ * Strips provided prefix and postfix from id and class attributes
+ *
+ * @param String htmlInput
+ * @param Object options
+ * @cb err, string
+ */
+function revertPurify(htmlInput, options, cb) {
```

Comment:
We're not really reverting the purify process here, instead we're reverting the namespaced classes and styles. Can you rename this to `revertNamespace`?